pyacq/core/tests/test_host.py

```diff
@@ -2,44 +2,28 @@
 import logging
 
 from pyacq.core.host import Host
-from pyacq.core.processspawner import ProcessSpawner
-from pyacq.core.rpc import RPCClient
 
 #~ logging.getLogger().level=logging.INFO
 
 
 def test_host1():
     
-    process_host0 = ProcessSpawner(Host, 'host0', 'tcp://127.0.0.1:*')
-    process_host1 = ProcessSpawner(Host, 'host1', 'tcp://127.0.0.1:*')
+    p1, host1 = Host.spawn('host1')
+    p2, host2 = Host.spawn('host2')
     
-    client0 = RPCClient('host0', process_host0.addr)
-    print('on ping: ', client0.ping())
-    
-    time.sleep(1.)
-    
-    process_host0.stop()
-    process_host1.stop()
-
+    ng11 = host1.create_nodegroup('ng1')
+    ng12 = host1.create_nodegroup('ng2')
+    ng21 = host2.create_nodegroup('ng3')
+    ng22 = host2.create_nodegroup('ng4')
 
-def test_host2():
-    
-    process_host0 = ProcessSpawner(Host, 'host0', 'tcp://127.0.0.1:*')
-    
-    client0 = RPCClient('host0', process_host0.addr)
+    assert len(host1.spawners) == 2
+    host1.close_all_nodegroups()
+    assert len(host1.spawners) == 0
+    host2.close_all_nodegroups()
+    assert len(host2.spawners) == 0
     
-    client0.create_nodegroup('nodegroup 0.1', 'tcp://127.0.0.1:*')
-    client0.create_nodegroup('nodegroup 0.2', 'tcp://127.0.0.1:*')
-    
-    time.sleep(2.)
-    
-    client0.close_nodegroup('nodegroup 0.1')
-    client0.close_nodegroup('nodegroup 0.2')
-    
-    time.sleep(1.)
-    
-    process_host0.stop()
-
+    p1.stop()
+    p2.stop()
 
 
 if __name__ == '__main__':
```

Comment:
I took it out because everything in that test was covered in the nodegroup and manager tests.